docs/installation.rst

```diff
@@ -105,18 +105,21 @@ Restart Apache:
 
 .. code-block:: bash
 
-  $ sudo service apache2 restart
+  $ sudo apache2ctl restart
```

Comment:
Why use ``apache2ctl`` rather than ``service``? I think ``service`` is the recommended way of interacting with system daemons on Debian (and it uses apache2ctl under the hood). Unless there's a good reason to bypass this, I think we should leave it as ``service``.